.github/workflows/bump_version.yml

```diff
@@ -16,14 +16,14 @@ jobs:
           token: ${{ secrets.DEDALO_PAT }}
       - name: Get next version
         id: tag_version
-        uses: mathieudutour/github-tag-action@v6.0
+        uses: mathieudutour/github-tag-action@v6.1
```

Comment:
Wouldn't it be better to use v6 without the minor version here?

Reply:
It doesn't exist :/